third_party/libtorch/BUILD

```diff
@@ -68,11 +89,18 @@ cc_library(
     hdrs = glob([
         'include/caffe2/**/*.h'
     ]),
-    srcs = [
-        'lib/libcaffe2_nvrtc.so',
-        'lib/libcaffe2_detectron_ops_gpu.so',
-        'lib/libcaffe2_observers.so',
-        'lib/libcaffe2_module_test_dynamic.so'
-    ],
+    srcs = select({
+        ":windows": [
+            'lib/caffe2_nvrtc.lib',
+            'lib/caffe2_detectron_ops_gpu.lib',
+            'lib/caffe2_module_test_dynamic.lib'
```

Comment:
Is libcaffe2_observers.lib missing in windows version?

Reply:
Yes, this does not exist in the release version I grabbed from pytorch.org.